demo/index.html

```diff
@@ -101,7 +101,7 @@ <h1 class="c-main__title">React Components</h1>
               <a class="u-fg-inherit" href="avatars">Avatars</a>
             </div>
             <div class="u-mb-sm">
-              <a class="u-fg-inherit" href="breadcrumbs">Breadcrumbs</a>
+              <a class="u-fg-inherit" href="storybook/?path=/story/components-breadcrumbs--default">Breadcrumbs</a>
```

Comment:
Since we have Accordions going to the README view, its probably best to do that here as well.

Reply:
I've updated the breadcrumbs and tabs examples to link to the `README`.